RELEASES.md

```diff
@@ -1,3 +1,202 @@
+Version 1.26.0 (2018-05-10)
+==========================
+
+Language
+--------
+- [Closures now implement `Copy + Clone` if all captured variables do.][49299]
+- [Stabilised the `!` type.][47630] (Also known as the never type).
+- [The inclusive range syntax e.g. `for x in 0..=10` is now stable.][47813]
+- [`impl Trait` is now stable allowing you to have abstract
+  return types.][49255] e.g. `fn foo() -> impl Iterator<Item=u8>` or
+  `fn open(path: impl AsRef<Path>)`.
+- [Pattern matching on references are now automatically dereferenced.][49394]
+- [128-bit integers in the form of `u128` & `i128` are now stable.][49101]
+- [`main` can now return `!`, `Result<(), E: Debug>`,
+  `Result<!, E: Debug>`][49162] In addition to `()`.
+- [Fixed entry slice patterns are now stable.][48516] e.g.
+  ```rust
+  let points = [1, 2, 3, 4];
+  match points {
+      [1, 2, 3, 4] => println!("All points were sequential."),
+      _ => println!("Not all points were sequential."),
+  }
+  ```
+- [Parentheses are now allowed in `dyn` traits.][48481]
```

Comment:
`dyn Trait` isn't stable yet, so should this go in the notes?

Reply:
I was wondering that myself but it was tagged with `relnotes` so I left it in.

Reply:
@Aaronepower @steveklabnik  The relnote was about a breaking change like this:

```rust
use std::ops::Fn as dyn;
fn g(_: Box<dyn(std::fmt::Debug)>) {}
```

Previously it compiles. Now it causes the E0658 "`dyn Trait` syntax is unstable" error.

Reply:
oh huh